examples/examples_supported_models.py

```diff
@@ -0,0 +1,22 @@
+from deepchecks.tabular.datasets.classification.iris import load_data, load_fitted_model
+from deepchecks.tabular.suites import model_evaluation
+
+ds_train, ds_test = load_data(data_format='Dataset')
```

Comment:
```suggestion
ds_train, ds_test = load_data(data_format='Dataset', as_train_test=True)
```
just prefer to have it explicitly